lib/binding.js

```diff
@@ -59,8 +67,15 @@ exports.fdatasync = function() {
 	throw new Error('Unimplemented');
 }
 
-exports.fstat = function() {
-	throw new Error('Unimplemented');
+exports.fstat = function(fd, req) {
+	const buf = new Buffer(STRUCT_LKL_STAT_SIZE);
```

Comment:
`new Buffer` as a way to create a buffer is deprecated. You can use either `Buffer.alloc()` or `Buffer.allocUnsafe()`